### DISCUSSION TITLE
Use of [1:] in `fmt.Println`

### DISCUSSION THREAD

**Discussion by ddritzenhoff** [benbjohnson/wtf#76] [Q&A] [answered]
Hey,

I just wanted to first and foremost thank you for the amazing code that you wrote. This has been my de-facto north star when creating a Go project myself.

I do have a question though!

1: What is the point of adding `[1:]` [here](https://github.com/benbjohnson/wtf/blob/05bc90c940d5f9e2490fc93cf467d9e8aa48ad63/cmd/wtf/dial.go#L57)? I tried removing it in the Go playground and saw absolutely no difference. My understanding is that you create one big string, which is just a slice of bytes, and you remove the first byte from that slice for some reason.

Cheers :)

**Comment by moorereason** [comment, marked as answer]
It's to remove the leading newline used to make the output line up nicely in the Go code...unlike this:

```go
	fmt.Println(`Manage WTF dials you own or are a member of.
Usage:
	wtf dial <command> [arguments]
The commands are:
	list        list all available dials
	create      create a new dial
	delete      remove an existing dial
	members     view list of members of a dial
	set         set your WTF level for a dial
`)
```
